packages/api-client/src/connections/mocks/MockEntityApi.ts

```diff
@@ -22,6 +25,13 @@ export class MockEntityApi implements EntityApiInterface {
     const { field, fields, filter } = queryOptions;
 
     let filteredEntities = foundEntities;
+    if (this.getAccessPolicy) {
```

Comment:
With the comment below about testing forwarded routes, we probably don't need the complexity here? Happy either way, just thinking no need to overcomplicate things if not necessary. Tests that need a permissions failure response could just mock it out specifically?

Reply:
> Tests that need a permissions failure response could just mock that response from api client specifically

Oh this is a great point I should definitely change it to just do that 😅 Got a bit too in my head about reusing the mock api client